### ISSUE TITLE
Prevent User to Add Agent

### ISSUE THREAD

**Issue by yarlanda** [Ylianst/MeshCentral#3165]
Hello,

Is it possible to prevent a user from downloading the Agent or add a new computer to Mesh Central?
Also the invitation can it be disabled for a user?

![image](https://user-images.githubusercontent.com/85727403/134852030-814212e6-1c16-4b2d-87c0-a96510de15a3.png)

Thanks

Yanick

**Comment by Ylianst** [comment]
I don't think that option is available, but I can add it. I am on vacation until October 6th.

**Comment by yarlanda** [comment]
Hello,

I would be grateful if you could do it.

Thanks for the heads up
